harness/determined/tensorboard/fetchers/__init__.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, Dict, List, Optional, Type
+
+from .azure import AzureFetcher
+from .base import Fetcher
+from .gcs import GCSFetcher
+from .s3 import S3Fetcher
+from .shared import SharedFSFetcher
+
+__all__ = [
+    "S3Fetcher",
+    "GCSFetcher",
+    "AzureFetcher",
+    "SharedFSFetcher",
+]
+
+_FETCHERS = {
+    "s3": S3Fetcher,
+    "gcs": GCSFetcher,
+    "azure": AzureFetcher,
+    "shared_fs": SharedFSFetcher,
+}  # type: Dict[str, Type[Fetcher]]
+
+
+def build(config: Dict[str, Any], paths: List[str], local_dir: str) -> Fetcher:
+    storage_config = config.get("checkpoint_storage", None)
+    if storage_config is None:
+        raise ValueError("config does not contain a 'checkpoint_stroage' key")
+
+    # XXX Fix me. Read the other code related to this.
```

Comment:
not sure what this means, but please resolve it

Reply:
I think I resolved but didn't remove message.